src/components/pages/adminpage.tsx

```diff
@@ -1,24 +1,84 @@
-import React from "react";
+import React, { useState } from "react";
 import Header from "../ui/header";
 import Sidebar from "../ui/sidebar";
+import EditButton from "../buttons/edit_button";
 
 const AdminPage: React.FC = () => {
-    return (
-        <div className="h-screen w-screen flex flex-col relative">
-            <div className="relative z-10 flex flex-col h-full">
-                <Header />
-                <div className="flex flex-1">
-                    <Sidebar />
-                    <main className="flex-1 p-4 text-black">
-                        <h1 className="text-3xl font-bold mb-4">Welcome to the Admin Page</h1>
-                    </main>
+  const [tableData, setTableData] = useState(
+    Array(20).fill(null).map(() => ({
+      position: "Position",
+      salary: "Salary",
+      gender: "Gender",
+      experience: "Experience",
+    }))
+  );
+
+  const handleSave = (index: number, updatedData: { position: string; salary: string; gender: string; experience: string }) => {
+    const newData = [...tableData];
+    newData[index] = updatedData;
+    setTableData(newData);
+  };
+
+  const handleDelete = (index: number) => {
+    const newData = tableData.filter((_, i) => i !== index);
+    setTableData(newData);
+  };
+
+  return (
+    <div className="h-screen w-screen flex flex-col relative">
+      <div className="relative z-10 flex flex-col h-full">
+        <Header />
+        <div className="flex flex-1">
+          <Sidebar />
+          <main className="flex-1 p-4 text-black">
+            <h1 className="text-3xl font-bold mb-4">Welcome to the Admin Page</h1>
+
+            {/* Tabel */}
+            <div className="flex flex-col items-center gap-4">
+              <div className="bg-white shadow-lg rounded-xl overflow-hidden max-w-full self-center mt-2">
+                <div className="overflow-auto max-h-96">
```

Comment:
```suggestion
                <div className="overflow-y-auto max-h-96">
```
Change to this, because we don't want horizotal scroll within the table, only vertical